docs/domains/Domain.json

```diff
@@ -343,6 +348,11 @@
             "IfNotPresent"
           ]
         },
+        "maxClusterConcurrentStartup": {
+          "description": "The maximum number of managed servers that the operator will start in parallel for a cluster, if maxConcurrentStartup is not specified at the cluster level. A default value of 0 means there is no configured limit.",
+          "type": "number",
```

Comment:
A default value of 0 means there --> A value of 0 (the default) means that there

Reply:
Ditto for other occurrences of the same wording.

Reply:
fixed